templates/ecthr_cases/alleged-violation-prediction/templates.yaml

```diff
@@ -0,0 +1,90 @@
+dataset: ecthr_cases
+subset: alleged-violation-prediction
+templates:
+  32404ed1-2276-401f-bb93-2937d9919585: !Template
+    id: 32404ed1-2276-401f-bb93-2937d9919585
+    jinja: '{{facts | join("\n")}}
+
+
+      These facts show potential violation of EHCR articles could have been violated.
+      The number of violated articles is
+
+      |||
+
+      {{{1:"one",2:"two",3:"three",4:"several"}[[4,labels | length] | min]}}'
+    name: implicit_advice_number
+    reference: implicitly ask for number of violated articles (rather than quantity)
+    task_template: false
+  3e5ba238-98ad-4d25-b84f-f226158ef8d6: !Template
+    id: 3e5ba238-98ad-4d25-b84f-f226158ef8d6
+    jinja: "The following facts relate to a claim brought before the European Court\
+      \ of Human Rights (ECtHR). \n{{facts | join(\"\\n\")}}\n\nQuestion: How many\
+      \ substantive articles in the ECHR could have been breached on these facts?\
+      \ If more than three substantive articles are breached, answer \"several\".\n\
+      Answer:\n|||\n{{{1:\"one\",2:\"two\",3:\"three\",4:\"several\"}[[4,labels |\
+      \ length] | min]}}"
+    name: ecthr_alleged_articles_declaration_at_end
+    reference: Explicitly ask question at end
+    task_template: false
+  7a56f5cc-20b5-4543-bb20-9c616d3f36dc: !Template
+    id: 7a56f5cc-20b5-4543-bb20-9c616d3f36dc
+    jinja: 'Question: Have one, two, three, or several articles of the ECHR been violated
+      on these facts?
+
+      {{facts | join("\n")}}
+
+
+      Answer:
+
+      |||
+
+      {{{1:"one",2:"two",3:"three",4:"several"}[[4,labels | length] | min]}}'
+    name: ecthr_alleged_articles_question_at_start
+    reference: Explicitly ask question at start of prompt
+    task_template: false
+  88825bbb-8cee-4be5-93e2-99be816e8469: !Template
+    id: 88825bbb-8cee-4be5-93e2-99be816e8469
+    jinja: '{{facts | join("\n")}}
+
+
+      These facts show potential violation of EHCR articles could have been violated.
+      The quantity of violated articles is
+
+      |||
+
+      {{{1:"one",2:"two",3:"three",4:"several"}[[4,labels | length] | min]}}'
+    name: implicit declaration
```

Comment:
The difference between this and `implicit_advice_number` is that I changed "number" to "quantity", but this might not be a helpful distinction, I'll remove one of them.